mesonbuild/backend/ninjabackend.py

```diff
@@ -515,7 +515,10 @@ def generate(self):
         ninja = environment.detect_ninja_command_and_version(log=True)
         if need_setup_vsenv:
             builddir = Path(self.environment.get_build_dir())
-            builddir = builddir.relative_to(Path.cwd())
+            try:
```

Comment:
This really deserves a comment, otherwise this looks good to me

Reply:
@rkuester What's the status of getting a comment here?

Reply:
Okay, took a stab at a comment. Thanks for the review.

Reply:
Thanks, I was mostly just concerned about someone well meaning removeing this try/except because "Why are we doing this here?" This looks good to me now, thanks.